Sources/Animators/TransitionAnimator/FoldAnimator.swift

```diff
@@ -58,6 +58,11 @@ extension FoldAnimator: UIViewControllerAnimatedTransitioning {
       return
     }
 
+    let (_, tempToViewController, _) = retrieveViewControllers(transitionContext: transitionContext)
```

Comment:
Because they are all the same, shall we create an `internal` extension method to `UIViewControllerAnimatedTransitioning` to share the code? `restoreToViewFrame` or some better method name?